slice/IceStorm/IceStorm.ice

```diff
@@ -36,148 +36,142 @@ module IceStorm
     /// A sequence of {@link LinkInfo} objects.
     sequence<LinkInfo> LinkInfoSeq;
 
-    /// This dictionary represents quality of service parameters.
+    /// Quality of service parameters.
     /// @see Topic#subscribeAndGetPublisher
     dictionary<string, string> QoS;
 
-    /// This exception indicates that an attempt was made to create a link that already exists.
+    /// The exception that is thrown when attempting to create a link that already exists.
     exception LinkExists
     {
         /// The name of the linked topic.
         string name;
     }
 
-    /// This exception indicates that an attempt was made to remove a link that does not exist.
+    /// The exception that is thrown when attempting to remove a link that does not exist.
     exception NoSuchLink
     {
         /// The name of the link that does not exist.
         string name;
     }
 
-    /// This exception indicates that an attempt was made to subscribe a proxy for which a subscription already exists.
+    /// The exception that is thrown when attempting to subscribe a proxy for which a subscription already exists.
     exception AlreadySubscribed
     {
     }
 
-    /// This exception indicates that an attempt was made to subscribe a proxy that is null.
-    exception InvalidSubscriber
```

Comment:
Removed because we no longer use this exception. It's also redundant with the null proxy we perform in various operations that do not throw a user exception when this check fails.